tests/generate-programs.py

```diff
@@ -23,6 +24,22 @@ def many_args(filename, op, num):
     with open(filename[:-4] + 'env', 'w+') as f:
         f.write('(200)')
 
+    if op.startswith('0x'):
+        hexop = op[2:]
+        if len(hexop) % 2 != 0: hexop = '0' + hexop
+        if len(hexop) > 2 or hexop[0] in 'fec8':
+            # in this case we need a length prefix for the atom
+            hexop = '8%x' % (len(hexop) // 2) + hexop
+    else:
+        hexop = KEYWORD_TO_ATOM[op].hex()
+
+    with open(filename[:-4] + 'hex', 'w+') as f:
+        f.write('ff02ffff01ff02ff06ffff04ff02ffff04ffff02ff04ffff04ff02ffff04ff05ffff'
+            '0180808080ffff0180808080ffff04ffff01ffff02ffff03ff05ffff01ff17ffff02'
+            'ff04ffff04ff02ffff04ffff11ff05ffff010180ffff0180808080ffff018300ffff'
+            '80ffff01ff01818080ff0180ff' + hexop +
+            ('ff05' * num) + '80ff018080')
+
```

Comment:
What does this do? Is there some way to make it more literate (or at least, comments)?

Reply:
it's the hex version of the clvm and chialisp code above. It's just that `run` can't compile it because it stack-overflows

Reply:
Can you add a comment about the run stack overflow?

Reply:
it's fairly well known: https://github.com/Chia-Network/clvm_tools/blob/develop/clvm_tools/cmds.py#L52